docs/source/developer_guide/coding_standards.rst

```diff
@@ -35,7 +52,55 @@ In addition, ensure that you:
    referenced, but most important, it prevents using a static analysis
    tool like pyflakes to automatically find bugs.
 
-.. _infrastructure::
+Setting up local code quality checks
+------------------------------------
+
+There are two options to set up local code quality checks:
+
+* using ``pre-commit`` for automated code formatting
+* setting up ``black``, ``flake8``, ``isort`` and/or ``numpydoc`` manually in a local dev IDE
+
+Using pre-commit
+^^^^^^^^^^^^^^^^
+
+To set up pre-commit, follow these steps in a python environment
+with the ``sktime`` ``dev`` dependencies installed:
+
+1. Install pre-commit:
+
+.. code:: bash
+
+   pip install pre-commit
```

Comment:
This dependency is already in the `dev` list. Why install it separately from `pip install -e .[dev]`?

Reply:
don´t know. This is verbatim what was in an existing section, which I just moved.

Reply:
updated